clonelists/Sony - PlayStation.json

```diff
@@ -2269,9 +2565,16 @@
 		"Rhapsody - A Musical Adventure": [
 			["Marl Oukoku no Ningyou-hime - The Adventure of Puppet Princess", 1]
 		],
+		"Ring of Sias": [
+			["SuperLite 1500 Series - Ring of Sias", 2]
+		],
 		"Risk - The Game of Global Domination": [
 			["Risk", 1]
 		],
+		"Rittai Ninja Katsugeki - Tenchu - Shinobi Gaisen": [
```

Comment:
This would be a `0` under _Tenchu - Stealth Assassins_. This means it will be selected if the user has the appropriate languages added to the filters.

```
"Tenchu - Stealth Assassins": [
			["Rittai Ninja Katsugeki - Tenchu - Shinobi Gaisen", 0],
			["Rittai Ninja Katsugeki - Tenchu", 1]
		],
```